contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -216,6 +218,8 @@ func NewSchedulerServer() *SchedulerServer {
 		kubeletEnableDebuggingHandlers: true,
 		containPodResources:            true,
 		nodeRelistPeriod:               defaultNodeRelistPeriod,
+		conntrackTCPTimeoutEstablished: 86400, // 1 day (1/5 default), copied from cmd/kube-proxy/app/server.go
```

Comment:
Do we need a default here? is there a way to detect it not being set (pointer?) and then not specify the flag later so that it uses the kube-proxy default?